sdk/tables/aztables/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Release History
+
+## 12.0.0-beta.1 (Unreleased)
```

Comment:
Shouldnt this start at v0.1.0? Since it'll be a new module, or am I missing some context here?

Reply:
Cross language we decided to synchronize on v12 as the starting point because it is the latest version when taking track1 into account across Comsos tables and storage tables

Reply:
The short answer is that aligning versions across languages isn't really an option for Go.  Longer answer, major versions have significant implications in Go, please see [semantic import versioning](https://github.com/golang/go/wiki/Modules#semantic-import-versioning) for the gory details.